include/awkward/Content.h

```diff
@@ -1,77 +1,107 @@
-// BSD 3-Clause License; see https://github.com/jpivarski/awkward-1.0/blob/master/LICENSE
+// BSD 3-Clause License; see
+// https://github.com/jpivarski/awkward-1.0/blob/master/LICENSE
 
 #ifndef AWKWARD_CONTENT_H_
 #define AWKWARD_CONTENT_H_
 
 #include <cstdio>
 
-#include "awkward/cpu-kernels/util.h"
 #include "awkward/Identity.h"
 #include "awkward/Slice.h"
+#include "awkward/cpu-kernels/util.h"
 #include "awkward/io/json.h"
 #include "awkward/type/Type.h"
```

Comment:
It looks like it's alphabetizing the `includes`. I used to do that my hand in Emacs because I could easily pipe it out to a shell command.

It would be more interesting if it could catch unused headers, which I probably have. In Python, I use fully qualified names everywhere, so I can easily check for unused imports by string-search. It's a much harder problem with C++ includes.

Removing unused headers could make the builds a little faster, which are a little uncomfortable at up to 2 minutes, since that impedes my favorite way of debugging.